docs/source/conf.py

```diff
@@ -17,8 +17,8 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "URLs-checker"
-copyright = "2020, urlstechie"
+project = "urlchecker"
+copyright = "2020-2022, urlstechie"
```

Comment:
Good catch here! Maybe we should eventually derive the ending year from datetime so we don't have to update ever again :P But don't worry about it for now.

Reply:
genius xD